src/middleware/normalize.rs

```diff
@@ -178,11 +199,11 @@ mod tests {
     }
 
     #[actix_rt::test]
-    async fn should_normalize_nothing_notrail() {
+    async fn should_normalize_notrail() {
         const URI: &str = "/v1/something";
 
         let srv = |req: ServiceRequest| {
-            assert_eq!(URI, req.path());
+            assert_eq!(URI.to_string() + "/", req.path());
```

Comment:
Testing the case mentioned in the issue #1536, a simple path gets trailing slash appended.